tests/unit/routes/rentals/show-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
An acceptance test section is currently being PR'd https://github.com/emberjs/guides/pull/1440
We may want to add a "show" test to the the acceptance tests either as part of or after this feature.

I think its fair to also test show route with a unit test, that the slug is being properly passed to the queryRecord call.  Should be fairly simple to add and will demonstrate route unit tests.